src/Parser.jl

```diff
@@ -323,7 +329,7 @@ end
 Parse an integer from the given bytes vector, starting at `from` and ending at
 the byte before `to`. Bytes enclosed should all be ASCII characters.
 """
-function int_from_bytes(pc::ParserContext{<:Any,IntType},
+function int_from_bytes(pc::ParserContext{<:Any,IntType,<:Any},
```

Comment:
I don't think this change (here or elsewhere) is strictly necessary (unspecified type parameters should default to `<:Any`), but it shouldn't hurt either.